Analytics/importers/attr_range_decorator.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime
+from typing import Callable, Any
+
+from models.attributes import Attributes
+from models.attribute_range import AttributeRange
+from app import create_app
+from db import db
+
+application = create_app()
+
+
+def update_attribute_ranges(func: Callable) -> Callable:
+    """
+    Function decorator, track minimum and maximum value of all attributes
+    :param func: Importer function
+    :return: Importer function wrapped with minimum and maximum value
+             functionality
+    """
+    def range_wrapper(*args: Any, **kwargs: dict):
+        """
+        Execute importer function and then compute minimum and maximum
+        values of attributes
+        :param args: Arguments of func parameter
```

Comment:
There are many words that can be short hand, but I don't believe up town func is one of them